packages/next/client/page-loader.js

```diff
@@ -101,6 +100,21 @@ export default class PageLoader {
     })
   }
 
+  getPageList() {
+    if (process.env.NODE_ENV === 'production') {
+      return this.promisedBuildManifest.then((buildManifest) =>
+        Object.keys(buildManifest)
+      )
+    } else {
+      // fetch fresh page list in development
+      return fetch(
+        `${this.assetPrefix}/_next/static/${this.buildId}/_devPagesManifest.json`
```

Comment:
Shouldn't the `buildId` always be `development` here to emphasize this code shouldn't work in production?

Reply:
Yeah we can make the buildId explicit here to stress that